projects/demo/src/app/stepper/stepper.demo.ts

```diff
@@ -53,7 +53,7 @@ export class StepperDemo {
   private getReactiveForm() {
     return new FormGroup({
       name: new FormGroup({
-        first: new FormControl('Luke', Validators.required),
+        first: new FormControl('', Validators.minLength(4)),
```

Comment:
I don't understand why this is changed?

Reply:
This is in order to have a demo of the support issue that we had initially (for testing purposes)